docs/get-started/examples.md

```diff
@@ -34,25 +34,26 @@ NOTE for contributors: This is a curated list. Here's what qualifies:
 
 Learn how leading teams build design systems.
 
-- [Workday Canvas](https://workday.github.io/canvas-kit/?path=/story/welcome-getting-started--page)
-- [Salesforce Lightning](http://design-system-react-components.herokuapp.com/?path=/story/sldsaccordion--base)
-- [IBM Carbon](https://react.carbondesignsystem.com/?path=/story/accordion--accordion)
-- [Shopify Polaris](https://main--5d559397bae39100201eedc1.chromatic.com)
-- [Airbnb Dates](http://airbnb.io/react-dates/?path=/story/daterangepicker-drp--default)
-- [Lonely Planet](http://lonelyplanet.github.io/backpack-ui/?path=/story/styles--design-tokens)
-- [Wix Style](https://www.wix-style-react.com/?path=/story/introduction-getting-started--getting-started)
+- [Airbnb Dates](https://airbnb.io/react-dates/?path=/story/daterangepicker-drp--default)
+- [AnyVision UI](https://storybook.anyvision.co/)
+- [AppNexus Lucid](https://appnexus.github.io/lucid/?path=/docs/documentation-introduction--introduction)
+- [Audi](https://react.ui.audi/)
```

Comment:
@JoshuaKGoldberg it seems that if you switch to HTTPS in the AnyVision UI design system it throws a 404:

This is what I got when I visited the page with HTTPS:
![anyvision-https](https://user-images.githubusercontent.com/22988955/133153657-7d2dcd2c-6040-4fa0-8502-f571f6d023e0.png)

And with HTTP:
![anyvision-http](https://user-images.githubusercontent.com/22988955/133153676-3f238e4e-51cd-44e0-92be-80f5a2f5bbac.png)


If you're ok with it, could you revert it back?

Reply:
Aha, darn, I could have sworn I checked them all!